GitUp/Application/AppDelegate.m

```diff
@@ -68,6 +65,20 @@ - (AboutWindowController *)aboutWindowController {
   return _aboutWindowController;
 }
 
+- (AuthenticationWindowController *)authenticationWindowController {
+  if (!_authenticationWindowController) {
+    _authenticationWindowController = [[AuthenticationWindowController alloc] init];
+    __weak typeof(self) weakSelf = self;
+    _authenticationWindowController.loadPlainTextAuthenticationFormKeychain = ^BOOL(NSURL * _Nonnull url, NSString * _Nonnull user, NSString *__autoreleasing *username, NSString *__autoreleasing *password) {
+      return [weakSelf.class loadPlainTextAuthenticationFormKeychainForURL:url user:user username:username password:password allowInteraction:YES];
+    };
+    _authenticationWindowController.savePlainTextAuthenticationToKeychain = ^(NSURL * _Nonnull url, NSString * _Nonnull username, NSString * _Nonnull password) {
+      [weakSelf.class savePlainTextAuthenticationToKeychainForURL:url withUsername:username password:password];
+    };
```

Comment:
We should probably just use the `[AppDelegate class]` call directly, Document.m does this already. You could also move these 2 methods into some Keychain class. Since there really is no dependency on AppDelegate, there isn't really a reason to have these callbacks.